src/query/processor/device.es.listener.ts

```diff
@@ -108,8 +108,8 @@ export class DeviceEsListener extends AbstractQueryEsListener {
         if (AbstractQueryEsListener.defined(event.description)) {
             locationDetails.description = event.description;
         }
-        if (Object.keys(locationDetails).length) {
-            deviceUpdate.locationDetails = locationDetails;
+        for (const [key, value] of Object.entries(locationDetails)) {
```

Comment:
Dit is wel cryptisch, vind ik. Wat stellen de key en value hier voor? Kunnen we daar geen betere namen voor verzinnen? (zodat ik het ook begrijp 😬 )

Reply:
De key en value stellen hier keys en values voor binnen het locationDetails object, dat zijn specifiek de naam, of de beschrijving van een device-locatie. Dat object maak ik daarboven aan, om dat daarna als genest object op te slaan binnen de database. Een andere benaming zou kunnen zijn locationDetailKey, locationDetailValue, maar dat verschil is mijns inziens triviaal.